test/hotspot/jtreg/runtime/ErrorHandling/SecondaryErrorTest.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2014, 2022 SAP SE. All rights reserved.
- * Copyright (c) 2014, 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2014, 2023, Oracle and/or its affiliates. All rights reserved.
```

Comment:
There are no longer any changes to this file in this patch so the
copyright year update should be backed out.